.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
```

Comment:
label: Creation Review, Plan Review, Progress Review, Release Review

Reply:
We settle on Release Review for now. If we need other reviews, we can then update the template.